arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-06-01",
```

Comment:
Is there a reason why the api-version does not have "-preview" on it? Given that you mentioned it's a preview service? (and that api-version is not being updated with all the changes happening in the spec?
@ravbhatnagar any comments on this?